packaging/windows/internal/cuda_install.bat

```diff
@@ -9,7 +9,13 @@ set SRC_DIR=%~dp0\..
 
 if not exist "%SRC_DIR%\temp_build" mkdir "%SRC_DIR%\temp_build"
 
-set /a CUDA_VER=%CU_VERSION:cu=%
+rem in unit test workflow, we get CUDA_VERSION, for example 11.1
```

Comment:
I'm probably missing something but could you explain why we need all these changes to the `packaging` directory?

As far as I understand the main issue was that the CircleCI machine had an outdated driver, but I feel like we should only need to change the `unittest`  jobs, not the packaging jobs. Or maybe this is also fixing another issue?